### PR TITLE
Rename aci_switch_association to aci_leaf_selector

### LINKED ISSUE #109: Rename aci_switch_association to aci_leaf_selector
Rename resource `aci_switch_association` to `aci_leaf_selector`.

Nic

### PR DESCRIPTION
**Pull request by Atul9**
closes #109

### CODE REVIEW

**Review by nkatarmal-crest — CHANGES_REQUESTED** [review]
Changes to docs are missing in this PR

**Comment by Atul9** [comment]
@nkatarmal-crest done, I have updated the PR

File: website/docs/d/infraleafs.html.markdown [hunk 1]
```diff
@@ -1,18 +1,18 @@
 ---
 layout: "aci"
-page_title: "ACI: aci_switch_association"
+page_title: "ACI: aci_leaf_selector"
 sidebar_current: "docs-aci-data-source-switch_association"
 description: |-
   Data source for ACI Switch Association
 ---
```
Comment (nkatarmal-crest) [inline website/docs/d/infraleafs.html.markdown:7]: sidebar_current and description must be updated

File: website/docs/r/infraleafs.html.markdown [hunk 2]
```diff
@@ -1,18 +1,18 @@
 ---
 layout: "aci"
-page_title: "ACI: aci_switch_association"
+page_title: "ACI: aci_leaf_selector"
 sidebar_current: "docs-aci-resource-switch_association"
 description: |-
   Manages ACI Switch Association
 ---
```
Comment (nkatarmal-crest) [inline website/docs/r/infraleafs.html.markdown:7]: sidebar_current and description must be updated

**Review by nkatarmal-crest — COMMENTED** [review]
aci.erb file is left to update resource and datasource name

File: website/docs/d/infraleafs.html.markdown [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 
-# aci_switch_association #
+# aci_leaf_selector #
 Data source for ACI Switch Association
```
Comment (Atul9) [inline website/docs/d/infraleafs.html.markdown:10]: @nkatarmal-crest should I rename this ^ too? `ACI Switch Association`

File: website/docs/d/infraleafs.html.markdown (hunk 1 above)
Comment (Atul9) [reply website/docs/d/infraleafs.html.markdown:7]: done, I have updated the PR

File: website/docs/r/infraleafs.html.markdown (hunk 2 above)
Comment (Atul9) [reply website/docs/r/infraleafs.html.markdown:7]: done, I have updated the PR

File: website/docs/d/infraleafs.html.markdown (hunk 3 above)
Comment (nkatarmal-crest) [reply website/docs/d/infraleafs.html.markdown:10]: yes and rename it into rdoc as well

**Comment by nkatarmal-crest** [comment]
resolve conflicts.

File: website/docs/d/infraleafs.html.markdown (hunk 3 above)
Comment (Atul9) [reply website/docs/d/infraleafs.html.markdown]: done, I have updated the PR

**Comment by Atul9** [comment]
@nkatarmal-crest done, I have resolved the merge conflict.

**Pull request merged**